ee/server/service/mdm.go

```diff
@@ -0,0 +1,89 @@
+package service
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+
+	"github.com/fleetdm/fleet/v4/pkg/fleethttp"
+	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	apple_mdm "github.com/fleetdm/fleet/v4/server/mdm/apple"
+	"github.com/micromdm/nanodep/client"
+	"github.com/micromdm/nanodep/storage"
+)
+
+func (svc *Service) GetAppleBM(ctx context.Context) (*fleet.AppleBM, error) {
+	if err := svc.authz.Authorize(ctx, &fleet.AppleBM{}, fleet.ActionRead); err != nil {
+		return nil, err
+	}
+
+	// if there is no apple bm config, fail with a 404
+	if !svc.config.MDM.IsAppleBMSet() {
+		return nil, notFoundError{}
+	}
+
+	appCfg, err := svc.AppConfig(ctx)
+	if err != nil {
+		return nil, err
+	}
+	tok, err := svc.config.MDM.AppleBM()
+	if err != nil {
+		return nil, err
+	}
+
+	appleBM, err := getAppleBMAccountDetail(ctx, svc.depStorage)
+	if err != nil {
+		return nil, err
+	}
+
+	// fill the rest of the AppleBM fields
+	appleBM.RenewDate = tok.AccessTokenExpiry
+	// TODO: default team will have to be set when https://github.com/fleetdm/fleet/issues/8733
+	// is implemented.
+	appleBM.DefaultTeam = ""
```

Comment:
This is noted in the ticket https://github.com/fleetdm/fleet/issues/8733, task number 4.